### DISCUSSION TITLE
Import from Apple Note

### DISCUSSION THREAD

**Discussion by maconly34** [glushchenko/fsnotes#1715] [Q&A]
Hi team,
Is it possible to import my folders and notes from the Apple Note application?
I don't want to spend days copying and pasting.
Thank's
Bruno

**Comment by dylanseeger** [comment]
Use Obsidian's free [Importer plugin](https://help.obsidian.md/import/apple-notes). It reliably creates individual Markdown files from the Apple Notes database. Copy those files into FSNotes and you're set.

**Reply by aleith** [reply]
can this Importer plugin be used as stand alone tool or do we need to install Obsidian to use the plugin?

**Reply by aleith** [reply]
well seeing as there's a Free version of Obsidian I used the plugin from within the app (probably the only way I guess).

now I have all the MD notes, but there's two folders: Apple Notes and Apple Note Attachments. Its not clear to me how i Import these to **FSNotes**, seeing as **FSNotes** has no import menu command. Do we copy the markdown files and attachments to somewhere in the **FSNotes** directory?

I found the markdown files created from Apple Notes by Obsidian and also managed to find the `On My Mac/FSNotes/` folder in finder (using the right-click "show this note in finder" contextual menu command…)

what next? just copy the files into a folder? How will the attachments link back to the notes without a formal import command?

That got the text in (some was clearly in HTML format though and not translating). Some lacked image attachments inline.

**Reply by glushchenko** [reply]
Obviously this plugin will not work for FSNotes.

**Reply by aleith** [reply]
it wasn't clear to me what the workflow would be, if the plugin worked as a "stand alone" or i needed to trial/buy Obsidian to use it within. 

i got somewhere and generated the structured MD files but had issues with images not "importing". it would seem FSNotes doesn't have a menu command for importing files, this kind of manual workaround is required, but some content and formatting isn't going to survive the process as far as i can tell.

**Comment by maconly34** [comment]
Hello,
thank's for your help !
I will try it

Merci
